core/excel_api.php

```diff
@@ -185,6 +190,12 @@ function excel_prepare_string( $p_value ) {
  * @return string
  */
 function excel_get_cell( $p_value, $p_type, array $p_attributes = array() ) {
+	if ( !is_int( $p_value ) ) {
+		$t_value = str_replace( array( '&', "\n", '<', '>' ), array( '&amp;', '&#10;', '&lt;', '&gt;' ), trim( $p_value ) );
```

Comment:
Is there still a need for the trim?
As mentioned before, I would expect that I get exactly the same data when exporting/importing.